rubin_sim/maf/metrics/incremental_template_metric.py

```diff
@@ -102,7 +111,13 @@ def run(self, data_slice, slice_point=None):
         idx_template_inputs = ok_template_input[: self.n_images_in_template]  # Images included in template
 
         Night_template_created = data_slice[self.night_col][idx_template_created]
-        N_nights_without_template = Night_template_created - data_slice[self.night_col][0]
+        N_nights_without_template = Night_template_created
```

Comment:
This seems incorrect - n_nights_without_template should not be the same as the night_template_created.

Reply:
n_nights_without_template could be counted from the start of the survey, or it could be counted from the first observation in this field. Either would be valid. 
Or it could be dropped, given that "night_template_created" is present, as well as n_images_until_template".

Reply:
Do you want to go ahead and drop it? I think this is reasonable.